src/open_inwoner/accounts/forms.py

```diff
@@ -119,12 +120,21 @@ class Meta:
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-        # make phonenumber required when 2fa-sms login is enabled
         config = SiteConfiguration.get_solo()
-        if not config.login_2fa_sms:
-            del self.fields["phonenumber"]
-        else:
-            self.fields["phonenumber"].required = True
+        self.fields["phonenumber"].required = bool(config.login_2fa_sms)
```

Comment:
`config.login_2fa_sms` is already a non-nullable bool, so should be able to just do:

```suggestion
        self.fields["phonenumber"].required = config.login_2fa_sms
```